services/infinispan-device-registry/src/main/java/org/eclipse/hono/registry/infinispan/InfinispanRegistryConfig.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.registry.infinispan;
+
+import org.eclipse.hono.deviceregistry.ApplicationConfig;
+import org.infinispan.manager.DefaultCacheManager;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.IOException;
+
+/**
+ * Spring Boot configuration for the Device Registry application.
+ *
+ */
+@Configuration
+public class InfinispanRegistryConfig extends ApplicationConfig {
+
+    @Value("${infinispan.config.file}")
```

Comment:
I'd rather have this defined as a property in a subclass of `org.eclipse.hono.config.ApplicationConfigProperties` ...

Reply:
Just to repeat that we I think there's a value in common config class for the registry implementations (that could extend ApplicationConfigProperties).